flash/image/segmentation/backbones.py

```diff
@@ -14,12 +14,17 @@
 from functools import partial
 
 from flash.core.registry import FlashRegistry
-from flash.core.utilities.imports import _TORCHVISION_AVAILABLE
+from flash.core.utilities.imports import _SEGMENTATION_MODELS_AVAILABLE, _TORCHVISION_AVAILABLE
 from flash.image.backbones import catch_url_error
 
 if _TORCHVISION_AVAILABLE:
     from torchvision.models import mobilenetv3, resnet
 
+if _SEGMENTATION_MODELS_AVAILABLE:
+    import segmentation_models_pytorch as smp
```

Comment:
Let's move `ENCODERS` to the bottom, so model imports and registry population are done together.